### PR TITLE
Update FAQ-journey.md

### PR DESCRIPTION
**Pull request by lcansby**
Updated information about 3PIP phones since these will be supported after 2023.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/FAQ-journey.md [hunk 1]
```diff
@@ -217,7 +217,8 @@ Teams meetings is now available on Surface Hub with Calling and Meetings experie
 
 ### Will current third-party IP (3PIP) phones continue to work with Microsoft Teams? And if so, how long?
 
-After the Skype for Business Online retirement date of July 31, 2021, users with 3PIP devices who have been migrated over to Teams only will be able to continue using their 3PIP devices with a limited set of functionalities until July 31, 2023.
+Skype for Business (3PIP) phones will be supported until further notice with new SIP phone support for Teams, so you can continue to use your existing Skype for Business phones as you move to Teams. Additionally, we will be supporting core calling features on SIP phones from Cisco, Yealink, Polycom, and others with Teams. Support for SIP phones will be available in the first half of 2021.
```
Comment (JohanFreelancer9) [inline Teams/FAQ-journey.md:220]:
```suggestion
Skype for Business (3PIP) phones will be supported until further notice with new SIP phone support for Teams, so you can continue to use your existing Skype for Business phones as you move to Teams. Additionally, we will be supporting core calling features on SIP phones from Cisco, Yealink, Polycom, and others with Teams. Support for SIP phones will be available in the first half of 2021. For more information see this [blog](https://www.microsoft.com/en-us/microsoft-365/blog/2020/08/03/take-communnext-level-microsoft-teams/).
```

File: Teams/FAQ-journey.md [hunk 2]
```diff
@@ -217,7 +217,8 @@ Teams meetings is now available on Surface Hub with Calling and Meetings experie
...
+https://www.microsoft.com/en-us/microsoft-365/blog/2020/08/03/take-communnext-level-microsoft-teams/
```
Comment (JohanFreelancer9) [inline Teams/FAQ-journey.md:221]:
```suggestion
```

**Comment by scanum** [comment]
@LanaChin PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by LanaChin** [comment]
Reached out to PM.

**Comment by LanaChin** [comment]
Hi @lcansby - thanks for your input.  We have updated that section with support info for 3PIP and SIP phones, and so am closing out your PR.

**Pull request closed without merging**